cmd/run.go

```diff
@@ -320,12 +320,12 @@ func run(_ *cobra.Command, _ []string) {
 		panic(err)
 	}
 
-	if err := runServer(context.Background(), config); err != nil {
+	if err := RunServer(context.Background(), config); err != nil {
 		panic(err)
 	}
 }
 
-func runServer(ctx context.Context, config *Config) error {
+func RunServer(ctx context.Context, config *Config) error {
```

Comment:
Does this need to be publicly exported? Is it intended to be used by the public Go community? Seems like this should stay private since it's primary usage isn't a reusable function but instead is just a helper to start the server in standalone mode (when running the server from the CLI `run` entrypoint).

Reply:
I want to use this from the test. What is the alternative?